resqpy/property/property_collection.py

```diff
@@ -15,13 +15,14 @@
 import resqpy.olio.uuid as bu
 import resqpy.olio.write_hdf5 as rwh5
 import resqpy.olio.xml_et as rqet
-import resqpy.time_series as rts
 from resqpy.olio.xml_namespaces import curly_namespace as ns
 
-from .property_kind import PropertyKind
 from .string_lookup import StringLookup
-from .property_common import same_property_kind, supported_property_kind_list, property_kind_and_facet_from_keyword, \
-    dtype_flavour, _cache_name, _cache_name_for_uuid, selective_version_of_collection, guess_uom
+from .property_common import dtype_flavour, _cache_name, _cache_name_for_uuid, selective_version_of_collection
+import resqpy.property._property_collection_create_xml_functions as pcxml
```

Comment:
Personally I have yoyo-ed between including the package identification in each module name, or not. At the moment I favour shorter file names without the package being repeated in each file name, so:
resqpy.property._create_xml_functions
instead of
resqpy.property._property_collection_create_xml_functions

But I see pro's and con's for both.

Reply:
I have trimmed the names somewhat, though leaving in the reference to 'collection' for now. Once the other property classes have been refactors I may revisit and remove the collection reference if possible